crates/turborepo-cache/src/async_cache.rs

```diff
@@ -163,11 +159,6 @@ impl AsyncCache {
         // Wait until flush callback is finished
         rx.await.ok();
     }
-
-    pub async fn shutdown(self) {
-        let Self { writer_thread, .. } = self;
-        writer_thread.await.unwrap();
-    }
```

Comment:
This was unused

Reply:
Hmm that feels wrong. If we don't await on the writer thread, there's a chance that we could exit before we finish adding stuff to the cache

Reply:
Yea it does - I can return this and track it outside of the async_cache struct to make sure we close it down.

Reply:
`wait` ensures that all workers have finished writing to the underlying cache. We could add a new `WorkerRequest::Shutdown` that does the same, but drops the receiving end so any future writes will fail.